content/en/platform/corda/5.0/deploying-operating/config/database-connection.md

```diff
@@ -25,4 +25,4 @@ Credentials can be encrypted. See [Configuration Secrets]({{< relref "./secrets.
 
 By default, connection details for the RBAC, Crypto, and virtual node databases are stored in the `db_connection` table of the configuration database and never published to the Kafka message bus. For more information about populating these values, see the [Manual Bootstrapping section]({{< relref "../deploying/bootstrapping.md#database" >}}).
 
-{{< enterprise-icon noMargin="true" >}} If you are using HashiCorp Vault as an external secret management system, you must ensure the passwords for the RBAC, Crypto, and virtual node databases are stored correctly in Vault. For more information, see [Encryption]({{< relref "../deploying/deploying.md#encryption">}}) in the _Deploying_ section. 
\ No newline at end of file
+{{< enterprise-icon noMargin="true" >}} If you are using HashiCorp Vault as an external secret management system, you must ensure the passwords for the RBAC, Crypto, and virtual node databases are stored correctly in Vault. For more information, see [Encryption]({{< relref "../deploying/deploying.html#external-secrets-service-hahahugoshortcode-s18-hbhb">}}) in the _Deploying_ section. 
```

Comment:
Is this intentional ?
`hahahugoshortcode-s18-hbhb`

Reply:
Yes, It's intentional. The icon for Enterprise functionality is added using Hugo shortcodes. Unfortunately, Hugo inserts a code for the shortcode when rendering the HTML. We were working with Zheng to see if we could find a fix but so far nothing. Hence, me testing the links this way so we have something working.